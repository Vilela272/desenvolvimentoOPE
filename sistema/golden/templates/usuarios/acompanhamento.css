.container{
    margin: 0 auto;
    width: 100%;
    max-width: 1080px;
    margin-top: 90px;
}

.dados_informacoes{
    margin: 0 auto;
    width: 100%;
    max-width: 1080px;
    margin-top: 90px;
}

.status_produto {
    display:flex;
    justify-content: space-evenly;
}

.status {
    background-color: #7E459B;
    color: #ffffff;
    text-align: center;
    padding: 20px;
    width: 20%;
}

.copyright {
    color: black;
    margin-top: 50px;
    text-align: center;
}