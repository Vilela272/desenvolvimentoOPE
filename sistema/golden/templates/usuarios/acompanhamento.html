{% extends 'base.html' %}
{% load static %}
{% block content %}
<!-- Preloader -->
<div id="preloader" >
    <i class="circle-preloader" ></i>
    <img src="{% static 'img/core-img/spinner.png' %}" alt="">
</div>

<!-- Stylesheet -->
<link rel="stylesheet" href="{% static 'site.css' %}">
<link rel="stylesheet" href="{% static 'botao.css' %}">


{% load static %}
<!-- ##### Start Payment ##### -->
<header class="header-area">

        <!-- Top Header Area -->
        <div class="top-header-area">
            <div class="container h-100">
                <div class="row h-100 align-items-center justify-content-between">
                    <!-- Breaking News -->
                    <div class="col-12 col-sm-6">
                        <div class="breaking-news">
                            <div id="breakingNewsTicker" class="ticker">
                                <ul>
                                    <li><a href="" style="color:black">Boas vindas!</a></li>
                                    <li><a href="" style="color:black">As melhores Blusas, Calças, Acessórios, Bonés e Cordões</a></li>
                                    <li><a href="" style="color:black">Você encontra aqui!</a></li>
                                    <li><a href="" style="color:black">GOLDEN BEAR COMPANY</a></li>
                                </ul>
                            </div>
                        </div>
                    </div>
        <!-- Top Social Info -->
        <div class="col-12 col-sm-6">
            <div class="top-social-info text-right">
                <a rel="noopener noreferrer" href="https://www.facebook.com/goldenbearcompany/" target="_blank"><i class="fa fa-facebook" aria-hidden="true" style="color:black"></i></a>
                <a rel="noopener noreferrer" href="https://instagram.com/goldenbearcompany?igshid=ztaa1r9rtdic" target="_blank"><i class="fa fa-instagram" aria-hidden="true" style="color:black"></i></a>
                <a href="{% url 'email' %}" target="_blank"><i class="fa fa-envelope" aria-hidden="true" style="color:black"></i>
                </a>
                <a href="" target="_blank" onclick="exibirTelefone()"><i class="fa fa-phone"aria-hidden="true" style="color:black"></i></a>
            </div>
        </div>
</div>
</div>
</div>

<!-- ##### Start Monitoring the product ##### -->
    <div class="container01">
    <div class="row">
        <div class="col-12">
            <div class="receipe-slider owl-carousel">
                <h1>Golden Bear Company</h1>
                <!--
                <img src="{% static 'img/core-img/golden.png' %}">
                -->
            </div>
        </div>
    </div>
</div>

<div class="container">
    <div class="status_produto">
        <a class="status">
            <img src="carrinho.png" alt="icone do carrinho" >
            <h3>Pedido Recebido</h3>
        </a>

        <a class="status">
            <img src="/usuarios/img/pagamento.png" alt="icone de pagamento" >
            <h3>Pagamento Realizado</h3>
        </a>

        <a class="status">
            <img src="/usuarios/img/aprovado.png" alt="icone aprovado" >
            <h3>Pagamento Aprovado</h3>
        </a>

        <a class="status">
            <img src="/usuarios/img/embalagem.png" alt="icone aprovado" >
            <h3>Preparado para Envio</h3>
        </a>

        <a class="status">
            <img src="/usuarios/img/enviado.png" alt="icone produto enviado" >
            <h3>Produto Enviado</h3>
        </a>

        <a class="status">
            <img src="/usuarios/img/presente.png" alt="icone finalizado" >
            <h3>Entrega Finalizada</h3>
        </a>

    </div>
    
</div>

<div class="dados_informacoes">
    <h3 class="titulo">Informações do pedido</h3>
    <ul class="dados_informacoes_lista">
        <li>Data de realização do pedido</li>
        <li>Forma de pagamento escolhida</li>
        <li>Status do pagamento</li>
        <li>Forma de envio escolhida</li>
        <li>Status do envio</li>
    </ul>
</div>

<div class="dados_informacoes">
    <h3 class="titulo">Dados para entrega</h3>
    <ul class="dados_informacoes_lista">
        <li>CEP</li>
        <li>Número</li>
        <li>Endereço</li>
        <li>Prazo de entrega</li>
        <li>Forma de Entrega</li>
    </ul>
</div>

<img src="carrinho.png" />
{% include 'partials/_footer.html' %}
{% endblock %}